### PR TITLE
Use absolute import instead of relative import for Python3

### PR DESCRIPTION
**Pull request by GeraldEV**

### CODE REVIEW

File: xcp/bootloader.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 import copy
 from typing import cast
 
-import branding  # pytype: disable=import-error
+import xcp.branding  # pytype: disable=import-error
```
Comment (psafont) [inline xcp/bootloader.py:33]: doesn't this fix the import error? looks like the comment can be dropped

Comment (GeraldEV) [reply xcp/bootloader.py]: You're right! I also need to update the references to it, I've pushed a change to fix both of those issues :)

**Review by psafont — APPROVED** [review]

**Comment by psafont** [comment]
The commit needs to be signed off:
```
git commit --amend --signoff
```

**Comment by liulinC** [comment]
I also fix this issue locally , and the pylint may also need to update

```
-import branding  # pytype: disable=import-error
+# branding module will be installed by other package
+import xcp.branding  # pylint: disable=import-error,no-name-in-module
```

**Pull request merged**